news/_posts/2021-12-14-prestashop-beyond-1-7.md

```diff
@@ -0,0 +1,105 @@
+
+##### 2. Renewed product page
+
+The all-new Back office product page that was introduced as experimental in 1.7.8.0 will be fully integrated in PrestaShop 8. This new page will provide merchants with an enhanced user experience and increased performance, especially when dealing with a large number of combinations. Developers will also benefit from a fully Symfony-based page, with extendable forms and reusable Commands and Queries.
+
+### A company neutral project
+
+PrestaShop is a community open source project, giving anyone the freedom to download it, use it, and modify to their needs. During the last few years, we have been working hard to clarify how an open source project like ours is different from "freemium product".
+
+We are committed to strengthening the community-built nature of the project. Last year, we welcomed our first two maintainers not affiliated with PrestaShop SA. In 2021, to better distinguish the project from the company who originally created it, we launched an independent website for the project: [prestashop-project.org](https://www.prestashop-project.org/).
+
+In 2022, our plan is to further differentiate the open source project from the homonymous company, by removing all company-related products and services from the project itself.
+
+PrestaShop 8 will no longer be bundled with the PrestaShop Addons marketplace, nor will it automatically download products from PrestaShop SA, nor will it depend on resources loaded from prestashop.com APIs. This represents a big leap forward in the project's neutrality, as we continue or work on an independent project governance.
+
+Of course, PrestaShop SA will continue to provide its products and services, for previous and new versions — but they will be distributed differently, as separate packages. These will likely be bundled in future products, which much like Linux distributions, will remain independent from the original open source project.
+
+Rest assured that contrary to other companies, who have chosen to abandon their open source platforms by treating them as spinoffs of their proprietary developments, PrestaShop SA will continue (and increase) its support to the open source project, and its future products will continue to be built _on top_ of the project, and not the other way around.
+
+### What happens with 1.7?
+
+PrestaShop 1.7.8 will follow the same support path as we described in the release roadmap above, meaning that it will receive:
+
+- Bug fixes until PrestaShop 8 is released
+- Critical and security fixes, if needed, for up to one year afterward.
+
+Upgrading from 1.7.8 to 8.0 should be significantly easier than upgrading from 1.6 to 1.7. Existing themes for 1.7 should remain fully compatible with this new version. While most modules that work with 1.7.8 should also work with PrestaShop 8 without any changes, some of them might need to be updated, in particular those affected by updated core libraries or PHP 8 compatibility.
```

Comment:
```suggestion
Upgrading from 1.7.8 to 8.0 should be significantly easier than upgrading from 1.6 to 1.7. Existing themes for 1.7 should remain fully compatible with this new version. While most modules that work with 1.7.8 should also work with PrestaShop 8 without any changes, some of them might need to be updated, particularly those affected by updated core libraries or PHP 8 compatibility.
```

> While most modules that work with 1.7.8 should also work with PrestaShop 8 without any changes

This is not really true. With all the deprecated code being removed from the core, I'm 100% sure that we can expect a lot of modules to be not compatible with v8. Remember that most of them are created to support 1.6 and 1.7 at the same time.

Reply:
How about we change "most" by "many"?

Reply:
"many" should be more accurate, but let's be honest, it's a major version, with some significant breaking changes, module developers will need to put a lot of work to make modules compatible, but we need them to do that :)